components/functions-library-mgt/org.wso2.carbon.identity.functions.library.mgt/src/test/java/org/wso2/carbon/identity/functions/library/mgt/FunctionLibraryManagementServiceTest.java

```diff
@@ -34,17 +32,14 @@
 import java.util.List;
 
 import static org.powermock.api.mockito.PowerMockito.when;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertNotNull;
-import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.*;
```

Comment:
As a practice, we avoid * imports. So, please revert this back to individual imports

Reply:
👍

Reply:
@pulasthi7 Thank you for your review.
I did change the import as per convention.

Reply:
Thanks @nitram509